### DISCUSSION TITLE
Graphgym config cfg.train.auto_resume not working?

### DISCUSSION THREAD

**Discussion by nilskre** [pyg-team/pytorch_geometric#7432] [Q&A] [answered]
Hello,

the setting is the training of a GraphGym model, especially starting training again from a saved checkpoint.

Expected behavior: By setting `cfg.train.auto_resume` to `true` the training process should check, if a saved model from before exists and continue training from this model/epoch.

Observed behavior: Training is again started from scratch (only the folder with the saved checkpoint is not deleted).

Can you reproduce this? 
Thanks in advance!

**Comment by rusty1s** [comment, marked as answer]
I am sorry for the late reply. I think this flag has been accidentally removed in a re-factor, I am really sorry. Let me try to bring it back.

**Reply by nilskre** [reply]
No worries, thanks for the explanation and confirmation.
